### PR TITLE
Add packages for R Course Week 4

### PR DESCRIPTION
**Pull request by jvtcl**
This PR adds the following packages needed to rebuild lessons in R Course Week 4 (see earthlab/eds-lessons-dev#124): 

- `cowplot`
- `maptools`'
- `ggsn` (there is no `conda-forge` feedstock for this one as of now -- I am currently using `devtools::install-github` to install it).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -92,6 +94,7 @@ RUN apt-get update \
 
 RUN ln -s /bin/tar /bin/gtar \ 
     && R --silent -e "devtools::install_github('earthlab/qtoolkit', dependencies = FALSE)" 
+    && R --silent -e "devtools::install_github('oswaldosantos/ggsn', dependencies = FALSE)"
```
Comment (lwasser) [inline Dockerfile]: @jvtcl did you try to build this locally. i'm getting a docker build error. i believe it needs a `\` to parse a new line. i'll add this directly now but please be sure to do that for the future!!

**Comment by lwasser** [comment]
@jvtcl it does look like this build. what lesson is ggsn used in? i'm just curious if we need it. installing from github is a bit dangerous so i'm thinking -- could we just provide an example of using it without actually building those chunks? Alternatively i could ask filipe about adding it to conda forge if it's a widely used R package. thoughts??

**Comment by jvtcl** [comment]
@lwasser all of these packages are just used in a single lesson: 

https://github.com/earthlab/eds-lessons-dev/blob/week-04-r-course-jt/courses/earth-analytics-r/04-vector-data-gis-r/review-materials/2017-07-25-plot02-custom-mapsggplot-R.Rmd

`ggsn` is used to add a north arrow and scale bar to the maps being made in the example. We could just set `eval = FALSE` to show the code only.

**Comment by lwasser** [comment]
@ocefpaf is it possible to get the ggsn r package on conda-forge? thank you!!

**Comment by ocefpaf** [comment]
> @ocefpaf is it possible to get the ggsn r package on conda-forge? thank you!!

See https://github.com/conda-forge/staged-recipes/pull/9011

**Pull request merged**